### DISCUSSION TITLE
Choose Column when using LinkToAnotherRecord

### DISCUSSION THREAD

**Discussion by stepanof** [nocodb/nocodb#2587] [Q&A]
Hi team.
I can't find solition in nocoDB for my issue, shall I create a Feature request or I can solve my issue using current nocoDB version?
Issue:
There are 3 tables - Region, City, Order. Regions has many Cities , Cities has many Orders
1) Create table 'Region' and add rows
2) Create table 'Cities' and add rows
3) For table 'Cities' create column "LinkToAnotherRecord", choose child table 'Region'
4) For each city choose 'Region' in created column. 
Now we have 2 linked to each other tables.

5) Create table 'Orders' and add rows
6) For table 'Orders' create Column "LinkToAnotherRecord", choose child table 'Region'
7) For each order choose 'Region' in created column
8) For table 'Orders' create Column named 'Cities' and ...

And here I want next column consist of the only cities which is associated with the choosen region!

Is it possible to do in current version?

Thankx for your help! :)

**Comment by stepanof** [comment]
@wingkwong 🙏🥺🙏
Anybody at home?...

**Reply by wingkwong** [reply]
Check this out - https://docs.nocodb.com/setup-and-usages/lookup

**Comment by elau7e** [comment]
There is question in the conditions from documentation example. Consider the model:
 **departments**->(has many)->**employees**->(has many)->**projects** (see departments, employees and projects)

Difficulties arise at the moment when it is necessary to select employees  for the project with same names ( namesake ). So, at company work 2 employee named PENELOPE, but in different departments (see penelope_departments). When considering the project, it is impossible to understand from which department the employee is (see project_choice-1,2). Can the following scenario be implemented? First, select the implementing department for the project, and then select employees only from the selected department?

![1-departments](https://user-images.githubusercontent.com/4416809/182875394-508e46f3-2395-45ce-a385-8c610c887b4c.png)
![2-employees](https://user-images.githubusercontent.com/4416809/182875399-d66ea618-42c1-4afa-8129-badaf6067320.png)
![3-projects](https://user-images.githubusercontent.com/4416809/182875405-48ab2c6c-d51d-46d9-87ad-6f99da233664.png)
![4-penelope_departments](https://user-images.githubusercontent.com/4416809/182875410-cd13a114-6dfa-4def-9277-313d7c8dcfbc.png)
![5-project_choice_1](https://user-images.githubusercontent.com/4416809/182875414-306d79ce-8dc1-41b9-8229-a5dbd34f2377.png)
![6-project_choice_2](https://user-images.githubusercontent.com/4416809/182875425-8c24e682-d0fd-44db-b266-e63437995889.png)
